nodejs/week3/controllers/contactsController.js

```diff
@@ -0,0 +1,25 @@
+const knex = require("../models/db")
+
+const contacts = async (req, res) => {
+    let query = knex("*").from("contacts");
+
+    if ("sort" in req.query && req.query.sort.includes("drop")) {
+      const orderBy = req.query.sort.toString();
+      if (orderBy.length > 0) {
+        query = query.orderByRaw(orderBy);
```

Comment:
I'd like to suggest to refactor this line of code, and use parameterized query to include the sort in query without sql injection vulnerability.
`const requestedSort = req.query.sort;`
`query = query.orderByRaw('?', [requestedSort]);`

Reply:
Thanks. I never came across this query. I will definitely try this. Looks clean and better.